pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarClientImplementationBindingImpl.java

```diff
@@ -327,7 +327,7 @@ public KeyValue<SchemaInfo, SchemaInfo> decodeKeyValueSchemaInfo(SchemaInfo sche
      * @return the jsonified schema info
      */
     public String jsonifySchemaInfo(SchemaInfo schemaInfo) {
-        return SchemaUtils.jsonifySchemaInfo(schemaInfo);
+        return SchemaUtils.jsonifySchemaInfo(schemaInfo, false);
```

Comment:
is it necessary to change the behavior in this PR? It seems that previously the output was pretty printed, but this changes it to be without pretty printing.

Reply:
Removed this change, I will start a discussion for it